### DISCUSSION TITLE
output directory missing in ShinyFMBN_3

### DISCUSSION THREAD

**Discussion by Better-L** [ep142/FoodMicrobionet#8] [Ideas]
Hi, this is to notify you that the output directory, together with all the subfolders for file export, are missing in ShinyFMBN_3.
All the best,
Luca B.

**Comment by ep142** [comment]
> Hi, this is to notify you that the output directory, together with all the subfolders for file export, are missing in ShinyFMBN_3. All the best, Luca B.

Hi Luca,
true. The issue, in case you want to know is that GitHub desktop for MacOS does not upload folders if they are empty, they should be fixed now (or will in a few minutes)

**Reply by Better-L** [reply]
Good, I can now see the folders on the Git-Hub web. Thanks!
